### PR TITLE
#[161382552] setup databse connection

### PR DESCRIPTION
**Pull request by Ruiru11**
#### What does this PR do?
. SetUp a databae connection
#### Description of Task to be completed?
. Connect to adatabse
#### What are the relevant pivotal tracker stories?
. https://www.pivotaltracker.com/story/show/161382552

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  - pip install -r requirements.txt
+  - pip install coveralls
+  - pip install pytest-cov
+
+script: 
+  -  pytest --cov=tests
+
+
+after_success:
+  - coveralls
```
Comment (ghost) [inline .travis.yml:15]: Add a new line here.

File: app/__init__.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+from app.models.database import Database_connection
+from app.views.products_views import don_item
+from app.views.sales_views import don_sale
+from app.views.users_views import don_user
+
+
+db = Database_connection()
+
+
```
Comment (ghost) [inline app/__init__.py:16]: Remove this space.

File: app/config.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    DATABASE_URL = 'postgres://postgres@localhost:5432/postgres/'
+
+
+#class TestingConfig(Config):
+ #   Postgresql = testing.postgresql.Postgresql()
+  #  DEBUG = True
+   # TESTING = True,
+    #print('>>>>>>>>>>>>>>>>>>', Postgresql.url())
+    #DATABASE_URL = Postgresql.url()
+
```
Comment (ghost) [inline app/config.py:23]: Why is this commented out?

Comment (Ruiru11) [reply app/config.py:23]: i had commented it out to test on something but i have rectified that its no longer commeted

**Pull request merged**